apps/web/src/design-system/button/Button.styles.ts

```diff
@@ -27,7 +27,9 @@ export const getOutlineStyles = (theme) => {
 
   return {
     border: '1px solid transparent',
-    background: 'transparent',
+    background: `${dark ? getGradient(colors.B17) : getGradient(colors.white)} padding-box, ${
+      colors.horizontal
+    } border-box`,
```

Comment:
fixed the bug with the `outline` button style which I introduced in the next branch